pkgs/development/python-modules/sanic/default.nix

```diff
@@ -87,6 +87,16 @@ buildPythonPackage rec {
   ] ++ lib.optionals stdenv.isDarwin [
     # https://github.com/sanic-org/sanic/issues/2298
     "test_no_exceptions_when_cancel_pending_request"
+  ] ++ lib.optionals (stdenv.hostPlatform.system == "aarch64-linux") [
+    "test_server_run"
+    "test_host_port"
+    "test_num_workers"
+    "test_access_logs"
```

Comment:
https://hydra.nixos.org/build/157553084/nixlog/1

I think at least some of them are related to timing errors, maybe because aarch64 is slower than x86_64 :thinking: (I don't know which machines are Hydra running for aarch64).

Reply:
do you mind saying a long the lines of, "these appear to be very sensitive to output of commands. output is different on aarch64"

Reply:
Done.

Reply:
@jonringer Any other issue? Can we get this merged?